### PR TITLE
Feature/core changes

### PR DESCRIPTION
**Pull request by martinfalk**
Changes:
* adjustments due to recent changes in core
* MolVisGL rasterizer update, based on PR https://github.com/inviwo/inviwo/pull/1498

### CODE REVIEW

File: molvis/molvisgl/src/processors/molecularrasterizer.cpp [hunk 1]
```diff
@@ -247,31 +246,36 @@ void MolecularRasterizer::process() {
...
     shader[ShaderType::Vertex]->setShaderDefine("FORCE_RADIUS", forceRadius_);
-
-    configureOITShader(shader);
     shader.build();
 }
 
 void MolecularRasterizer::configureLicoriceShader(Shader& shader) {
+    Rasterizer::configureShader(shader);
+
     utilgl::addDefines(shader, lighting_);
```
Comment (petersteneteg) [inline molvis/molvisgl/src/processors/molecularrasterizer.cpp:279]: lighting comes from the renderer.

File: molvis/molvisgl/src/processors/molecularrasterizer.cpp [hunk 2]
```diff
@@ -247,31 +246,36 @@ void MolecularRasterizer::process() {
...
     }
 }
 
+/*
+void SphereRasterizer::configureShader(Shader& shader) {
+    Rasterizer::configureShader(shader);
+    utilgl::addDefines(shader, labels_, periodic_, config_, clip_);
+    shader.build();
+}
+*/
```
Comment (petersteneteg) [inline molvis/molvisgl/src/processors/molecularrasterizer.cpp:266]: remove? that about periodic/config/clip?

File: molvis/molvisgl/include/inviwo/molvisgl/processors/molecularrasterizer.h [hunk 3]
```diff
@@ -40,32 +39,40 @@
...
 
 public:
     MolecularRasterizer();
     virtual ~MolecularRasterizer() = default;
 
-    virtual void process() override;
-
     virtual void initializeResources() override;
 
+    virtual void preprocess() override;
```
Comment (petersteneteg) [inline molvis/molvisgl/include/inviwo/molvisgl/processors/molecularrasterizer.h:66]: Might as will just remove the final from process and override that one

File: molvis/molvisgl/include/inviwo/molvisgl/processors/molecularrasterizer.h [hunk 4]
```diff
@@ -80,17 +87,18 @@ class IVW_MODULE_MOLVISGL_API MolecularRasterizer : public Processor {
     const float BallAndStickVDWScale = 0.3f;
     const float BallAndStickLicoriceScale = 0.5f;
 
+    //virtual void configureShader(Shader& shader) override;
```
Comment (petersteneteg) [inline molvis/molvisgl/include/inviwo/molvisgl/processors/molecularrasterizer.h:90]: remove

**Review by petersteneteg — CHANGES_REQUESTED** [review]

File: molvis/molvisgl/src/processors/molecularrasterizer.cpp (hunk 2 above)
Comment (martinfalk) [reply molvis/molvisgl/src/processors/molecularrasterizer.cpp:266]: copied from `void SphereRasterizer::configureShader()` for reference. Should be removed.

**Pull request merged**